file-formats/intf/zif_aff_intf_v1.intf.abap

```diff
@@ -1,37 +1,54 @@
 INTERFACE zif_aff_intf_v1 PUBLIC.
 
-  TYPES: BEGIN OF component,
-           name        TYPE zif_aff_types_v1=>object_name_30,
-           description TYPE zif_aff_types_v1=>description_60,
-         END OF component,
-         components    TYPE SORTED TABLE OF component WITH UNIQUE KEY name,
-         subcomponents TYPE SORTED TABLE OF component WITH UNIQUE KEY name,
+  TYPES:
+    "! Component description (e.g., for methods, attributes, types)
+    BEGIN OF component_description,
+      "! Name of the component
+      name        TYPE zif_aff_types_v1=>object_name_30,
+      "! Description of the component
+      description TYPE zif_aff_types_v1=>description_60,
+    END OF component_description,
+    "! Interface component descriptions
+    component_descriptions    TYPE SORTED TABLE OF component_description WITH UNIQUE KEY name,
+    "! Sub component descriptions (e.g., for parameters or exceptions)
+    subcomponent_descriptions TYPE SORTED TABLE OF component_description WITH UNIQUE KEY name,
 
-         BEGIN OF method,
-           name        TYPE zif_aff_types_v1=>object_name_30,
-           description TYPE zif_aff_types_v1=>description_60,
-           parameters  TYPE subcomponents,
-           exceptions  TYPE subcomponents,
-         END OF method,
-         methods TYPE SORTED TABLE OF method WITH UNIQUE KEY name,
+    "! Method description
+    BEGIN OF method.
+      INCLUDE TYPE component_description. TYPES:
```

Comment:
heh, not seen this before

but, https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#no-more-than-one-statement-per-line

Reply:
Actually, formatted structures with includes this way the very first time. ;)
I don't like to add the TYPES in the beginning, because you can't see the structure very well. 
But I also agree, multiple statements per line aren't nice. I can change it

Reply:
INCLUDE TYPE messes everything up 😄 note how the initial suggestion https://github.com/SAP/abap-file-formats/pull/70/commits/fa6f62ce62effe7a46e0290400035a9f4d4fbf64 did not have any INCLUDE TYPE, some parts can be refactored to avoid INCLUDE TYPE, others are compromises

Reply:
> INCLUDE TYPE messes everything up 😄

Agree. I am still not sure, what I prefer...